authentication/models.py

```diff
@@ -98,26 +112,71 @@ class Company(models.Model):
     startup_idea = models.TextField(blank=True)
     tags = models.CharField(max_length=255, blank=True)
 
+    @classmethod
+    def get_company(cls, *args, **kwargs):
+        return cls.objects.get(**kwargs)
+    
+    @classmethod
+    def get_companies(cls, *args, **kwargs):
+        return cls.objects.filter(**kwargs)
+    
+    def get_company_type(self):
+        if self.is_startup == True:
+            return STARTUP
+        else: 
+            return INVESTMENT
+    
+    def get_info(self):
+        fields = ('brand', 'common_info', 'contact_phone', 'contact_email')
+        startup_fields = ('product_info', 'startup_idea')
+        data = {}
+        company_type = self.get_company_type()
+        data['company_type'] = company_type
+        if company_type == STARTUP:
+            for k in startup_fields:
+                data[k] = self.__getattribute__(k)
```

Comment:
Since this method isn't classmethod, we assume that the instance of class has all the needed attributes; even if the required field is blank, instance would return None. Nevertheless, it's really useful to have additional insurance. I have added AttributeError handling so now if it raises None will be returned as value of the attribute in question.